internal/util/csiconfig.go

```diff
@@ -95,9 +100,9 @@ func readClusterInfo(pathToConfig, clusterID string) (*ClusterInfo, error) {
 			err, string(content))
 	}
 
-	for _, cluster := range config {
-		if cluster.ClusterID == clusterID {
-			return &cluster, nil
+	for i := range config {
```

Comment:
this looks unrelated to NFS, please remove and create its own PR

Reply:
creating a different commit, linter seems to be catching this error